src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.h

```diff
@@ -78,6 +78,8 @@ template <class dtype>
 struct DataItemHandle {
   CkArrayIndex arrayIndex;
   const dtype *dataItem;
+
+  DataItemHandle(dtype* _ptr) : dataItem(_ptr) {}
```

Comment:
Maybe just reverse the order of the parameters of the other constructor and add a default parameter for `_idx`?